packages/shared/styles/color.ts

```diff
@@ -53,18 +53,12 @@ const getLuminance = (luminanceVal1: number, luminanceVal2: number) => {
   return (l1 + 0.05) / (l2 + 0.05);
 };
 
-// TODO: when/if we start using rgb/rgba colors in
-// design-tokens, write getHexContrast function that can get
-// rgb/rgba contrast as well
```

Comment:
These two are 4 years old. I do not see us using rgb/rgba colors in the foreseeable future. I don't think this is a great TODO to leave for future devs without context or a task.